Api/DocumentGeneration.BFF.API.csproj

```diff
@@ -0,0 +1,37 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <Nullable>enable</Nullable>
+    <ImplicitUsings>enable</ImplicitUsings>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Npgsql.EntityFrameworkCore.PostgreSQL" Version="8.0.2" />
+    <PackageReference Include="Swashbuckle.AspNetCore" Version="6.4.0" />
+	<PackageReference Include="Asp.Versioning.Http" Version="8.0.0" />
+	<PackageReference Include="Asp.Versioning.Mvc.ApiExplorer" Version="8.0.0" />
+	<PackageReference Include="Microsoft.AspNetCore.Authentication.JwtBearer" Version="8.0.1" NoWarn="NU1605" />
+	<PackageReference Include="Microsoft.AspNetCore.Authentication.OpenIdConnect" Version="8.0.1" NoWarn="NU1605" />
+	<PackageReference Include="Microsoft.AspNetCore.Mvc.ApiExplorer" Version="2.1.3" />
+	<PackageReference Include="Microsoft.AspNetCore.Mvc.Core" Version="2.1.38" />
+	<PackageReference Include="Microsoft.AspNetCore.OpenApi" Version="8.0.1" />
+	<PackageReference Include="Microsoft.Identity.Web" Version="2.15.2" />
+	<PackageReference Include="Microsoft.Identity.Web.DownstreamApi" Version="2.15.2" />
+	<PackageReference Include="Serilog" Version="3.1.1" />
+	<PackageReference Include="Serilog.AspNetCore" Version="8.0.1" />
+	<PackageReference Include="Serilog.Enrichers.AssemblyName" Version="1.0.9" />
+	<PackageReference Include="Serilog.Enrichers.Environment" Version="2.3.0" />
+	<PackageReference Include="Serilog.Enrichers.Process" Version="2.0.2" />
+	<PackageReference Include="Serilog.Enrichers.Thread" Version="3.1.0" />
+	<PackageReference Include="Serilog.Extensions.Hosting" Version="8.0.0" />
+	<PackageReference Include="Serilog.Settings.Configuration" Version="8.0.0" />
+	<PackageReference Include="Swashbuckle.AspNetCore" Version="6.4.0" />
```

Comment:
11 and 29 both have AspNetCore?
And the indentation is a bit funky. It runs though :)